src/libopensc/pkcs15-sec.c

```diff
@@ -754,7 +757,8 @@ int sc_pkcs15_compute_signature(struct sc_pkcs15_card *p15card,
 		r = modlen;
 	}
 
-	sc_mem_clear(buf, sizeof(buf));
+err:
+	sc_mem_secure_free(buf, buflen);
```

Comment:
would it make sense to keep the `sc_mem_clear()` here to zeroize the memory before freeing?